src/components/common/Button/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Text, View, TouchableOpacity } from 'react-native';
+import { func, string } from 'prop-types';
+import styles from './styles';
+
+const Button = ({ disabled, handleSubmit, textTitle }) => (
+  <View style={styles.container}>
+    <TouchableOpacity
+      onPress={handleSubmit}
+      style={[styles.buttonContainer, disabled ? styles.btnContainerDisable : null]}
```

Comment:
if the second argunment is null then it's preferable if you use
```suggestion
      style={[styles.buttonContainer, disabled && styles.btnContainerDisable]}
```